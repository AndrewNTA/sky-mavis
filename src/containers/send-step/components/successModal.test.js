import React from 'react';
import renderer from 'react-test-renderer';
import SuccessModal from './successModal';

test('SendStep should render success modal', () => {
  const component = renderer.create(
    <SuccessModal
      currency={'EUR'}
      onConfirm={() => {}}
    />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});