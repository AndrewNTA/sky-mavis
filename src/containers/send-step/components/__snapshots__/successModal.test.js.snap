// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SendStep should render success modal 1`] = `
<div
  className="success-modal"
>
  <div
    className="sm-overlay"
  />
  <div
    className="sm-main"
  >
    <div
      className="sm-title"
    >
      Successfully sent
    </div>
    <div
      className="sm-content"
    >
      <div>
        <span>
          Your
        </span>
        <strong
          className="sm-currency"
        >
          EUR
        </strong>
        <span>
          has been sent!
        </span>
      </div>
      <div>
        Thank you for using our service
      </div>
    </div>
    <button
      className="r-primary-btn full"
      onClick={[Function]}
    >
      OK
    </button>
  </div>
</div>
`;
